settings/src/components/account-status.js

```diff
@@ -65,7 +65,7 @@ export default function AccountStatus() {
 					bodyText={
 						webAuthnEnabled
 							? 'You have two-factor authentication enabled using security keys.'
-							: 'You have not registered any security keys.'
+							: 'You have not enabled security keys for two-factor authentication.'
```

Comment:
You can have security keys registered but not have the provider enabled, so this is more accurate, and also more consistent with the TOTP message [below](https://github.com/WordPress/wporg-two-factor/pull/233/files#diff-8a81bc73a741dacb0ff448a75ab1f1fa1809145d97c1f89b6706a4e2da0c4a9fR81).